ReactWindows/ReactNative/Common/AccessibilityHelper.cs

```diff
@@ -0,0 +1,255 @@
+using ReactNative.UIManager;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+
+namespace ReactNative.Common
+{
+    /// <summary>
+    /// Accessibility helper methods.
+    /// </summary>
+    public static class AccessibilityHelper
```

Comment:
Does this need to be public? If so, please add comments for the <param> and <returns> docs below on public APIs.

Reply:
Comments added.

---
In reply to: [164202649](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r164202649) [](ancestors = 164202649)